src/uproot/version.py

```diff
@@ -0,0 +1,19 @@
+# file generated by setuptools_scm
+# don't change, don't track in version control
```

Comment:
Don't check this file into version control; 3816ead34084b0f21bbed8aae9344b4559c47f69 adds a check against that.